testhelpers/docker_registry.go

```diff
@@ -17,15 +17,16 @@ import (
 )
 
 type DockerRegistry struct {
-	Host            string
-	Port            string
-	Name            string
-	server          *httptest.Server
-	DockerDirectory string
-	username        string
-	password        string
-	regHandler      http.Handler
-	authnHandler    http.Handler
+	Host             string
+	Port             string
+	Name             string
+	server           *httptest.Server
+	DockerDirectory  string
+	username         string
+	password         string
+	regHandler       http.Handler
+	authnHandler     http.Handler
+	customPrivileges map[string]*ImagePrivileges
```

Comment:
This is a map from an `imageName` to its permissions, right?
I think it's worth adding it as a comment

Reply:
Added in the following [commit](https://github.com/buildpacks/imgutil/pull/125/commits/714d0d197edf3ff7391210592d4c3552cdfca10b)